tekton-catalog/any-sequencer/cmd/root.go

```diff
@@ -418,6 +433,46 @@ func watchRun(labelSelector string, tasks []string, failedTasksCh chan string) {
 	}
 }
 
+func watchSkippedTasks(fieldSelector string, tasks []string, skippedTasksCh chan string) []string {
```

Comment:
I'm not sure but since we already watched pr, can we get the run status/tr status from it to reduce use multiple watches. 
Also I saw the code said 
```
        // list of tasks that were skipped due to when expressions evaluating to false
	// +optional
	SkippedTasks []SkippedTask json:"skippedTasks,omitempty"
```
So I'm not sure whether it's only for "when" but not for task failed. If it's this case then if tekton wait all running task done then it will not done.

Anyway it's good now, please just try these cases, thanks.

Reply:
@pugangxa Great comments, yeah, we should only use pipelinerun watcher. Updated, pls take a another look, thanks.